crates/lib/tests/include_sass.rs

```diff
@@ -3,5 +3,5 @@
 fn basic() {
     let css: &str = grass::include!("./input.scss");
 
-    assert_eq!(css, "a{color:red}");
+    assert_eq!(css, "a{color:red;width:calc(100% + 32px)}");
```

Comment:
If possible, I'd like for the `include_sass` tests to only test features specific to the macro. Since this is intended to test compressed output, just having the 1 test should be fine.

```suggestion
    assert_eq!(css, "a{color:red}");
```

Reply:
I also removed it from the input